ddev/hatch.toml

```diff
@@ -10,6 +10,7 @@ e2e-env = false
 dependencies = [
   "pyyaml",
   "vcrpy",
+  "deepdiff",
```

Comment:
why do you need deepdiff in the `ddev` testing environment? 😂

Reply:
oh yeah I have it in 2 places now, will remove.